src/http/builtin_http_calls.cpp

```diff
@@ -77,6 +77,11 @@ namespace dsn {
         .with_callback(
             [](const http_request &req, http_response &resp) { update_config(req, resp); })
         .with_help("Updates the value of a config");
+
+    register_http_call("listAllConfigs")
```

Comment:
```suggestion
    register_http_call("configs")
```

RESTful API emphasize the "resource" rather than the "action" in the path. 

`/configs` is clear enough that shows this URL is for displaying all configs.